src/test/encoding/check-generated.sh

```diff
@@ -24,16 +24,21 @@ for type in `./ceph-dencoder list_types`; do
 	    continue
 	fi
 
-	./ceph-dencoder type $type select_test $n dump_json > $tmp1
-	./ceph-dencoder type $type select_test $n encode decode dump_json > $tmp2
-	./ceph-dencoder type $type select_test $n copy dump_json > $tmp3
-	./ceph-dencoder type $type select_test $n copy_ctor dump_json > $tmp4
+	./ceph-dencoder type $type select_test $n dump_json > $tmp1 & pids="$!"
+	./ceph-dencoder type $type select_test $n encode decode dump_json > $tmp2 & pids="$pids $!"
+	./ceph-dencoder type $type select_test $n copy dump_json > $tmp3 & pids="$pids $!"
+	./ceph-dencoder type $type select_test $n copy_ctor dump_json > $tmp4 & pids="$pids $!"
+	for pid in $pids; do wait $pid; done
```

Comment:
So for me the code is safe as wait will assert if one pid return failed and will not get 127 because of races between process.